app/views/pages/dashboard.html.erb

```diff
@@ -0,0 +1,62 @@
+<% content_for :page_title do %>
+  What's available in Find
+<% end %>
+
+<h1 class="heading-large">
+  What's available in Find right now
+</h1>
+
+<div class="grid-row">
+  <div class="column-one-third">
+    <div class="data">
+      <span class="data-item bold-xlarge"><%= number_with_delimiter(@datasets_count) %></span>
+      <span class="data-item bold-xsmall">datasets</span>
```

Comment:
Although an edge case, this would be an improvement:

```erb
<span class="data-item bold-xsmall"><%= pluralize(@datasets_count, 'dataset') %></span>
```